src/hooks/async.js

```diff
@@ -1,4 +1,4 @@
-import { useEffect, useState } from 'react';
+import { useEffect, useState, useCallback } from 'react';
```

Comment:
There are obvious bugs here. Please do not merge

Reply:
@mikeattara we need to do away with the async hook. To get the loading effect you need you can do it effectively in the action dispatch for fetch events instead